check/checkSpeciesTags.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+import re
+from copy import copy
+from pathlib import Path
+from sys import argv
+
+re_tags = re.compile(r"^ *tags *= *\[")
+re_start_effects = re.compile(r"^ *effectsgroups *= *\[")
+re_start_foci = re.compile(r"^ *foci *= *\[")
+re_end_list = re.compile(r"^ *\]")
+re_macro = re.compile(r"^ *\[\[([A-Z_]+)\]\]")
+# Commented out are not used by the AI, most likely they never will.
+# For tags that are implemented as species effects, only the AI
+# is using the tags.
+skills_to_check = [
+    "INDUSTRY",
+    "RESEARCH",
+    "INFLUENCE",
+    # "STOCKPILE",
+    "POPULATION",
+    "HAPPINESS",
+    "SUPPLY",
+    "OFFENSE_TROOPS",
+    # "DEFENSE_TROOPS",
+    "WEAPONS",
+    "DETECTION",
+    "STEALTH",
+    "FUEL",
+]
+foci_to_check = [
+    "INDUSTRY",
+    "RESEARCH",
+    "INFLUENCE",
+]
+
+
+def check_skill(name):
+    if (
+        name.startswith("AVERAGE")
```

Comment:
https://docs.python.org/3/library/stdtypes.html?highlight=str%20startswith#str.startswith

> prefix can also be a tuple of prefixes to look for. 

You could simplify this code.

Reply:
You live and learn... :)
I will update it.